com.unity.netcode.gameobjects/Runtime/Messaging/MessageHeader.cs

```diff
@@ -11,11 +11,12 @@ internal struct MessageHeader
         /// unchanged - if new messages are added or messages are removed, MessageType assignments may be
         /// calculated differently.
         /// </summary>
-        public byte MessageType;
+        public uint MessageType;
```

Comment:
Is this a required change for this PR?

Reply:
It's not required, no, but I thought it made sense. Making the other one larger meant it made sense to start packing the header. If the header's packed, then there's no reason to limit the number of messages to the byte range. It still ends up serialized as a byte due to packing.